blog-meeting-minutes/2024-01-19-community-hour.md

```diff
@@ -1,9 +1,9 @@
 ---
-slug: office-hour-19-01-2024
-title: Office hour 19.01.2024
+slug: community-office-hour-2024-01-19
+title: Community Office hour 2024-01-19
 authors: 
     - harald_zierer
-tags: [tractus-x-office-hour, meeting-minutes, community]
+tags: [community-office-hour, meeting-minutes, community]
```

Comment:
In the end they are all just tag used to filter the meeting minutes. The more the better... 😄 

The idea for the "community tag" was to group "our" meeting minutes vs. more official one like release management. 
Same applies to "community" vs. "meeting-minutes" for e.g. project management announcements

@stephanbcbauer will there be that kind of minutes/announcements in the future? If not, I would remove the superfluous tags.

Reply:
@hzierer at least there is nothing planned like that, but you never know ;) But since we don't have it right now, I think it's OK to remove it. Thanks